src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2122 @@
+            }
+        }
+
+        /// <summary>
+        /// Disconnects the voice control service.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        /// <pre>
+        /// The state must be ready.
+        /// </pre>
+        /// <post>
+        /// The state must be initialized.
+        /// </post>
+        public static void Unprepare()
+        {
+            ErrorCode error = VcMgrUnprepare();
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "Unprepare Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Retrieves all the supported languages.
+        /// A language is specified as an ISO 3166 alpha-2 two letter country-code
+        /// followed by ISO 639-1 for the two-letter language code.
+        /// For example, "ko_KR" for Korean, "en_US" for American English.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <returns>
+        /// List of strings for supported languages.
+        /// </returns>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to an invalid state.
+        /// 2) This exception can be due to operation failed.
+        /// </exception>
+        /// <exception cref="NotSupportedException">This exception can be due to not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        /// <pre>
+        /// The state must be ready or initialized.
+        /// </pre>
+        public static IEnumerable<string> GetSupportedLanguages()
+        {
+            _supportedLanguages = new List<string>();
+            _supportedLanguagesDelegate = (IntPtr language, IntPtr userData) =>
```

Comment:
use local variable if it is a sync method

Reply:
Changed to use local variable.

Reply:
_supportedLanguagesDelegate it also

Reply:
Fixed.